### DISCUSSION TITLE
DeepLX custom token support

### DISCUSSION THREAD

**Discussion by littleblack111** [immersive-translate/immersive-translate#1380] [Help]
### 概述 | Summary

DeepLX is usable without token for the server. But for security reasons. I would love to be using some sort of authorization like the one DeepLX supported - token
This shouldn’t be hard to implement since it’s just a breezier header

**Comment by theowenyoung** [comment]
You can implement token verification using URL search parameters, for example:

https://xxxx.com/translate?toke=xxxxx
